wiki/Help_centre/Tournament_bans/en.md

```diff
@@ -24,15 +24,15 @@ This includes situations like:
 
 - As above, any use of third-party cheating tools or utilities to gain an advantage
 - Making use of multiple accounts to enter a tournament
-- Attempting to evade a silence or infringement by using another account
+- Attempting to evade a silence or other penalty by using another account
```

Comment:
I don't get the change. Yes, even if semantically it is making reference to the violation, that is precisely the point. Evading the violation itself is an issue, just as much as evading a previous punishment is. Do not feel like it needs to be changed.

Reply:
op is right: infringement/violation is not the same penalty/consequence, and "evading violations" doesn't semantically make sense (unless you mean avoiding making violations, which of course wouldn't be punished)